src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -9,6 +9,9 @@
 
 namespace Microsoft.ML
 {
+    /// <summary/>
+    /// Collection of extension methods for the <see cref="DataOperationsCatalog"/>.
```

Comment:
Well, this is one "Collection of extension methods for the <…. catalog> to read from delimited text files such as CSV TSV"